File: features/login/impl/presentation/src/androidDeviceTest/kotlin/com/mockdonalds/app/features/login/presentation/WelcomeStateRobot.kt
package com.mockdonalds.app.features.login.presentation

import com.mockdonalds.app.core.test.StateRobot

class WelcomeStateRobot : StateRobot<WelcomeUiState, WelcomeEvent>() {

    override fun defaultState() = WelcomeUiState(
        eventSink = createEventSink(),
    )
}

File: iosApp/iosAppTests/UIComponent/Login/WelcomeStateRobot.swift
import ComposeApp

final class WelcomeStateRobot: BaseStateRobot<WelcomeUiState, WelcomeEvent> {

    override func defaultState() -> WelcomeUiState {
        WelcomeUiState(
            eventSink: createEventSink()
        )
    }
}
